images/cilium/Dockerfile

```diff
@@ -110,3 +122,14 @@ ARG TARGETARCH
 COPY --from=builder /go/bin/dlv /usr/bin/dlv
 RUN mv /usr/bin/cilium-agent /usr/bin/cilium-agent-bin
 COPY images/scripts/debug-wrapper.sh /usr/bin/cilium-agent
+
+#
+# Cilium debug symbols image.
+#
+FROM scratch as debug-symbols
```

Comment:
Is it better to inherit from the already build cilium release image, or is it better to start from scratch (and only have the debug symbols within the image)?
Both work from a Datadog perspective. From a cilium perspective, the latter might make it easier to consume the image for some users ?

Reply:
I think we could possibly kill this one off completely.

We deliver a debug image, and I’m now (well, pending checking my code works) adding the debug symbol files if the binaries were stripped to the debug image. So you now either have non-stripped binaries, or you have stripped binaries + symbol files alongside them (same directory though per your other comment /usr/lib/debug might be better) in the debug image.

Pros:
- No extra debug-symbols image to deliver, can probably easily sell this to upstream.

Cons:
- You need to figure out if the binaries were stripped or not and decide which files to process.
- You don’t have a clear indication which files are worth processing since there’s a complete OS.

Reply:
This is gone now, I deliver the debug symbols in the `debug` image.